sql/parser/create.go

```diff
@@ -75,7 +75,7 @@ func (p *Parser) parseIfNotExists() (bool, error) {
 	return true, nil
 }
 
-func (p *Parser) parseTableConfig(cfg *database.TableConfig) error {
+func (p *Parser) parseFieldsConstraints(info *database.TableInfo) error {
```

Comment:
```suggestion
func (p *Parser) parseFieldConstraints(info *database.TableInfo) error {
```

Reply:
I don't agree because this method parses _all_ fields constraints and not only one. Wdyt?

Reply:
"Field constraint" is a compound word. When you write compound words in plural form, only the last one takes an `s`.

Example: one train station, two train stations, one car park, two car parks

📖 https://www.myenglishpages.com/site_php_files/grammar-lesson-plural-compound-nouns.php